site/topics/numbers/number-bases.rst

```diff
@@ -418,6 +418,301 @@ Hexadecimal (Base 16)
+
+
+* In general, multiply the value of the digit in base ten by the value of the base in base ten to the power of its corresponding position
+
+* For example, given some number of :math:`n` digits :math:`d` in base :math:`b`
+
+    :math:`(d_{n-1}d_{n-2}...d_{2}d_{1}d_{0})_{b}`
+
+
+* The value in base ten would be calculated as follows
+
+    :math:`(d_{n-1} \times b^{n-1} + d_{n-2} \times b^{n-2} + ... + d_{2} \times b^{2} + d_{1} \times b^{1} + d_{0} \times b^{0})_{10}`
+
+
+* Consider the number :math:`J204_{20}` --- note that :math:`J_{20} == 19_{10}` in this example
+
+
+    :math:`(19 \times 20^{3} + 2 \times 20^{2} + 0 \times 20^{1} + 4 \times 20^{0})_{10}`
+
+    :math:`(19 \times 8000 + 2 \times 400 + 0 \times 20 + 4 \times 1)_{10}`
+
+    :math:`(152000 + 800 + 0 + 4)_{10}`
+
+    :math:`152804_{10}`
+
+
+
+Converting from Decimal
+-----------------------
+
+* To convert a number from decimal to some arbitrary base :math:`b`
+
+    * Divide the number by :math:`b` and keep the quotient :math:`q_{0}` and remainder :math:`r_{0}`
+    * The value of :math:`r_{0}` is the digit for the least significant position
+    * Divide :math:`q_{0}` by the base :math:`b` to get a new quotient :math:`q_{1}` and remainder :math:`r_{1}`
+    * The value of :math:`r_{1}` is the digit for the next position
+    * Repeat until the quotient is zero
+
+
+* Consider converting the number :math:`123_{10}` to binary
+
+    .. list-table:: Converting :math:`123_{10}` to Binary
+        :widths: 50 50 50 50
+        :header-rows: 1
+
+        * - Digit/Bit Position
+          - Division
+          - :math:`q`
+          - :math:`r`
+        * - :math:`0`
+          - :math:`123/2`
+          - :math:`61`
+          - :math:`1`
+        * - :math:`1`
+          - :math:`61/2`
+          - :math:`30`
+          - :math:`1`
+        * - :math:`2`
+          - :math:`30/2`
+          - :math:`15`
+          - :math:`0`
+        * - :math:`3`
+          - :math:`15/2`
+          - :math:`7`
+          - :math:`1`
+        * - :math:`4`
+          - :math:`7/2`
+          - :math:`3`
+          - :math:`1`
+        * - :math:`5`
+          - :math:`3/2`
+          - :math:`1`
+          - :math:`1`
+        * - :math:`6`
+          - :math:`1/2`
+          - :math:`0`
+          - :math:`1`
+
+
+* Therefore, the number :math:`123_{10}` is :math:`1111011_{2}`
+
+* Consider converting the number :math:`123_{10}` to hexadecimal
+
+    .. list-table:: Converting :math:`123_{10}` to Hexadecimal
+        :widths: 50 50 50 50
+        :header-rows: 1
+
+        * - Hex Digit Position
+          - Division
+          - :math:`q`
+          - :math:`r`
+        * - :math:`0`
+          - :math:`123/16`
+          - :math:`7`
+          - :math:`11` or :math:`B`
+        * - :math:`1`
+          - :math:`7/16`
+          - :math:`0`
+          - :math:`7`
+
+
+* Therefore, the number :math:`123_{10}` is :math:`7B_{2}`
+
+    * Remember, :math:`B_{16} == 11_{10}`
```

Comment:
remove, prefer to add subscripts to the divisions above (line 573)

Reply:
changed, 

There could be an argument to add the subscripts to everything in the table, but I don't actually think that is necessary as I do not feel that there is any ambiguity. The subscripts are not addressing ambiguity; the subscripts are reminding the reader of the fact that B is 11.

Reply:
The _Remember..._ line can be removed still.

Reply:
Odd... I swear I did. 

I blame the ctrl-z ing I did to undo the previous changes to find like 573.